gtsfm/averaging/rotation/shonan.py

```diff
@@ -56,23 +66,65 @@ def run(self, num_images: int, i2Ri1_dict: Dict[Tuple[int, int], Optional[Rot3]]
 
         for (i1, i2), i2Ri1 in i2Ri1_dict.items():
             if i2Ri1 is not None:
-                between_factors.append(
-                    BetweenFactorPose3(
-                        i2,
-                        i1,
-                        Pose3(
-                            i2Ri1,
-                            np.zeros(
-                                3,
-                            ),
-                        ),
-                        noise_model,
-                    )
-                )
+                # ignore translation during rotation averaging
+                i2Ti1 = Pose3(i2Ri1, np.zeros(3))
+                between_factors.append(BetweenFactorPose3(i2, i1, i2Ti1, noise_model))
 
         obj = ShonanAveraging3(between_factors, shonan_params)
 
         initial = obj.initializeRandomly()
         result_values, _ = obj.run(initial, self._p_min, self._p_max)
 
-        return [result_values.atRot3(i) for i in range(num_images)]
+        wRi_list_consecutive = [None] * num_connected_nodes
```

Comment:
I dont see the need to name the variable as `consecutive`?

Reply:
I guess I didn't want to confuse the reader if they saw the same variable name in `run()` and `__run_with_consecutive_ordering()`, since they symbolize very different quantities

Reply:
cool